coalib/results/TextPosition.py

```diff
@@ -31,3 +31,50 @@ def line(self):
     @property
     def column(self):
         return self._column
+
+    def __le__(self, other):
+        """
+        Test whether ``self`` is behind the other ``TextPosition``.
```

Comment:
ah I would add "... is behind **or equals** the other ...". Same for `__ge__` below :3